workflows/genome-assembly/bacterial-genome-assembly/bacterial_genome_assembly-tests.yml

```diff
@@ -11,8 +11,8 @@
   outputs:
     shovill_contigs_graph:
       assert:
```

Comment:
You need to use 'asserts' instead of 'assert' everywhere see https://github.com/galaxyproject/iwc/pull/530/files#diff-82a68e44fab78fc5a709d0d4ca32c7a288c5772847d4151f62994e2261c106e8

Reply:
Done :smiley: